ios/RNCAsyncStorage.m

```diff
@@ -34,6 +34,27 @@
   }
 }
 
+static BOOL setBackupEnabled(NSString* path, BOOL isEnabled)
+{
+    NSFileManager *fileManager = [[NSFileManager alloc] init];
+    
+    BOOL isDir;
+    BOOL exists = [fileManager fileExistsAtPath:path isDirectory:&isDir];
+    BOOL success = false;
+    
+    if(isDir && exists) {
+        NSURL* pathUrl = [NSURL fileURLWithPath:path];
+        NSError *error = nil;
+        success = [pathUrl setResourceValue:[NSNumber numberWithBool:isEnabled] forKey:NSURLIsExcludedFromBackupKey error: &error];
```

Comment:
Nit: You can use the short-hand notation instead of `+[NSNumber numberWithBool:isEnabled]`:
```suggestion
        success = [pathUrl setResourceValue:@(isEnabled) forKey:NSURLIsExcludedFromBackupKey error:&error];
```